### ISSUE TITLE
Infiltrating Voltarus quest in Zul'Drak is bugged.

### ISSUE THREAD

**Issue by therinrik** [dalaranwow/dalaran-wow#5523]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The teleporter to Voltarus does not work, even when wearing the ghoul disguise.

**Expected behaviour**: Tell us what should happen instead
The player stands in the center of the platform while wearing the ghoul disguise and is teleported to Voltarus.

**Steps to reproduce the problem**:

1. Equip ghoul costume
2. Stand on the platform
3. 

**Include proofs for this behaviour**

https://imgur.com/a/TXaOYxK

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Quest ID: 12661

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by boujii** [comment]
This is one of those 
https://i.imgur.com/C0cc9e2.png

This is probably pulled from the gameobject_questitem table. 
I'll try and research into this more once a GM confirms that it is in fact not working.

**Comment by Golganeth** [comment]
The teleporter to Voltarus works as intended while wearing the Scourge disguise.
